stable_baselines3/common/vec_env/__init__.py

```diff
@@ -19,8 +19,10 @@
 if typing.TYPE_CHECKING:
     from stable_baselines3.common.type_aliases import GymEnv
 
+VecEnvWrapperT = TypeVar("VecEnvWrapperT", bound=VecEnvWrapper)
```

Comment:
not sure to understand the difference with before, but if it works...

Reply:
```python
T = TypeVar("T")

def func(arg: T) -> T: ...
```

ensures that the return type of func is the same as the arg type.

Before, `unwrap_vec_wrapper` was allowed to return any subtype of `VecEnvWrapper`, regardless of the type of `vec_wrapper_class`.